lib/mini_profiler/profiler.rb

```diff
@@ -586,19 +586,22 @@ def ids_comma_separated(env)
     def get_profile_script(env)
       path = "#{env['RACK_MINI_PROFILER_ORIGINAL_SCRIPT_NAME']}#{@config.base_url_path}"
 
+      position = @config.position.match("-") ? @config.position.split("-") : ["top", @config.position]
```

Comment:
I think its way cleaner just to have Rack::MiniProfiler::Config handle this. A lot easier to test and can amend vertical / horizontal when you set position.